api/workAllocation/constants/actions.ts

```diff
@@ -4,7 +4,7 @@ const ASSIGN = {
 };
 const CANCEL = {
   id: 'cancel',
-  title: 'Cancel task'
+  title: 'Cancel task',
```

Comment:
What are these extra commas for? Auto-formatter or something?

Reply:
Linting errors I was getting. These are occurring on wa-integration branch as well so will fix that I believe

Reply:
Yeah the Lint is dumb. Unfortunately its required

Reply:
A trailing comma is _required_? That's absurd!